Lib/queue.py

```diff
@@ -234,6 +234,13 @@ def _put(self, item):
 
     def _get(self):
         return heappop(self.queue)
+    
+    def __repr__(self):
+        max_length_to_display = 20
+        if len(self.queue) > max_length_to_display:
+            return "PriorityQueue(" + str(self.queue[:max_length_to_display - 1]) + " ..... " + str(self.queue[-1]) + ")"
```

Comment:
I think you could use the repr here `repr(self.queue[:max_length_to_display - 1])` and others

Reply:
`PriorityQueue` should also be `self.__class__.__name__`